tests/test_get_files.py

```diff
@@ -117,12 +117,12 @@ def test_get_files_zipped_product(s3_stubber: Stubber):
     ]
     stub_list_files(s3_stubber, 'myJobId', 'myBucket', files)
     stub_get_object_tagging(s3_stubber, 'myBucket', 'myJobId/myProduct.zip', 'product')
+    stub_expiration(s3_stubber, 'myBucket', 'myJobId/myProduct.zip')
```

Comment:
I'd suggest we remove stubbing the log files from `test_get_files_zipped_product` and `test_get_files_zipped_product`, and add a brand now test case for `test_get_files_failed_job` that has only a log file stubbed